<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsWindow</class>
 <widget class="QWidget" name="StatisticsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="filterLabel">
       <property name="text">
        <string>Filter:</string>
       </property>
       <property name="buddy">
        <cstring>filterLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="filterLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="detailLabel">
       <property name="text">
        <string>Count:</string>
       </property>
       <property name="buddy">
        <cstring>attributeComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="attributeComboBox">
       <item>
        <property name="text">
         <string>cat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>root</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="resultsTable">
      <property name="selectionBehavior">
       <enum>SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="percentageCheckBox">
       <property name="text">
        <string>Show Percentages</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="totalHitsLabel">
       <property name="toolTip">
        <string>Total values found</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="filterProgress">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>filterLineEdit</tabstop>
  <tabstop>attributeComboBox</tabstop>
  <tabstop>resultsTable</tabstop>
  <tabstop>percentageCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
